components/rating/metadata/rating.yml

```diff
@@ -262,7 +262,7 @@ examples:
     name: Quiet (read-only)
     markup: |
       <div class="spectrum-Rating spectrum-Rating--quiet is-readOnly">
-        <input class="spectrum-Rating-input" type="range" min="0" max="5" value="0" readOnly aria-label="Rating">
+        <input class="spectrum-Rating-input" type="range" min="0" max="5" value="0" readOnly disabled aria-label="Rating">
```

Comment:
I see why this has to be disabled but will this make the whole component hidden from screen readers in the Read-only variant?

Reply:
Yeah, I wasn't sure about this... The `readonly` attribute does nothing at all, so I think I'll have to remove `disabled` and reset the value in JS... But not really important since this is just for demo purposes...

Reply:
> I updated the enhancement to reset the value on change for readonly ratings and removed `disabled` from the readonly examples. I think this should do the trick.

Looks good to me!